Makefile

```
@@ -232,3 +228,10 @@ test-e2e:
 .PHONY: test-doc-automation
 test-doc-automation:
 	$(RUN_GINKGO) $(GINKGO_FLAGS_ONE) --junit-report="test-doc-automation.xml"  tests/documentation/...
+
+
+# Generate OpenAPISpec library based on ododevapispec.yaml inside pkg/apiserver-gen; this will only generate interfaces
+# Actual implementation must be done inside pkg/apiserver-impl
+.PHONY: generate-apiserver
+generate-apiserver: ## Generate OpenAPISpec library based on ododevapispec.yaml inside pkg/apiserver-gen; also go fmt the files
+	podman run --rm -v ${PWD}:/local openapitools/openapi-generator-cli generate -i /local/ododevapispec.yaml -g go-server -o /local/pkg/apiserver-gen --additional-properties=outputAsLibrary=true,onlyInterfaces=true && echo "Formatting generated files:" && go fmt ./pkg/apiserver-gen/...
```

I have to add the `:Z` option to the volume mount to be able to access the volume on my system. @rm3l @valaparthvi Do you also have this restriction on your system?

```suggestion
	podman run --rm -v ${PWD}:/local:Z openapitools/openapi-generator-cli generate -i /local/ododevapispec.yaml -g go-server -o /local/pkg/apiserver-gen --additional-properties=outputAsLibrary=true,onlyInterfaces=true && echo "Formatting generated files:" && go fmt ./pkg/apiserver-gen/...
```

---

It worked for me without ":Z"